File: src/c/problem set 1/lesson 3 - mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_block(int row);

int main(void)
{
    int size;
    do
    {
        size = get_int("Size of Pyramid: ");
    }
    while (size < 1);

    for (int row = 1; row <= size; row++)
    {
        for (int spaces = size - row; spaces > 0; spaces--)
        {
            printf(" ");
        }
        print_block(row);
        printf("  ");
        print_block(row);
        printf("\n");
    }
}

void print_block(int row)
{
    for (int column = 1; column <= row; column++)
    {
        printf("#");
    }
}

File: src/python/problem set 6/lesson 2 - sentimental-mario-more/mario.py
from cs50 import get_int


while True:
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        break


for i in range(n):
    blocks = i + 1
    spaces = n - blocks
    print(" " * spaces, end="")
    print("#" * blocks, end="")
    print("  ", end="")
    print("#" * blocks)
